src/content/en/ilt/pwa/offline-quickstart.md

```diff
@@ -55,7 +55,7 @@ Caching data locally results in speed and cost benefits for mobile users (many o
 
 ### Access to browser independent features
 
-Service workers are the foundation for browser independent features for web applications. Because a service worker's lifecycle is independent of the web app's lifecycle, the service worker can take actions even when the web app isn't running (for example, receiving push notifications, syncing data in the background, and geofencing). Combined with progressive enhancement, these features can be safely added to your app without breaking it in unsupported browsers. To see if a target browser supports a given service worker feature, check  [Is Service Worker Ready?](https://jakearchibald.github.io/isserviceworkerready/)
```

Comment:
This comes from a Google doc - will send you the link and ask that you work with the owner to update it there, in the mean time, would you please remove this file from the PR, otherwise the changes will be reverted if we update again.